testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.tests.js

```diff
@@ -18167,6 +18167,31 @@ QUnit.test('Pressing symbol keys inside detail grid editor does not change maste
     assert.deepEqual(this.keyboardNavigationController._focusedCellPosition, { rowIndex: 0, columnIndex: 1 }, 'Master grid focusedCellPosition is not changed');
 });
 
+QUnit.test('Should open master detail by click if row is edited in row mode (T845240)', function(assert) {
```

Comment:
Need to add tests for the different kind of 'startEditAction'

Reply:
Done